beancount_dkb/ec.py

```diff
@@ -141,4 +142,11 @@ def _read_meta(fd):
                             )
                         )
 
+                # Closing Balance
+                meta = data.new_metadata(file_.name, 0)
+                entries.append(
+                    data.Balance(meta, self._date_to, self.account,
```

Comment:
Is there a case when the dates mentioned in the `Kontostand vom` and `Bis` metadata entries could be different? I ask to double check if there's a need to get the date out of the `Kontostand vom...` metadata to use in the `data.Balance` call. Purely to ensure correctness.

Reply:
My understanding is that the two _should_ always be the same but there may well be edge cases I'm unaware of.

While I had considered parsing the balance date out of the "Kontostand vom …" line, I decided against doing so. Relying on parsing a free text string actually felt more fragile to me than assuming that the balance given is always that of the statement's end date.